### PR TITLE
New version: 8x8.VirtualOfficeDesktop version 8.7.2

### PR DESCRIPTION
**Pull request by PckgrBot**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123035)

### CODE REVIEW

File: manifests/8/8x8/VirtualOfficeDesktop/8.7.2/8x8.VirtualOfficeDesktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{E5A87C4F-C56C-43F4-A174-E1033A02EDCD}'
+AppsAndFeaturesEntries:
+- UpgradeCode: '{66F8B350-94E0-43AF-9D26-C2385B310C96}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://vod-updates.8x8.com/ga/work-64-msi-v8.6.3-1.msi
+  InstallerSha256: 81DB508FEE1F21D724EB8709ACF79D0BD4171B72D0E3F2E02C44C150E72AE924
```
Comment (shivaabhishek07) [inline manifests/8/8x8/VirtualOfficeDesktop/8.7.2/8x8.VirtualOfficeDesktop.installer.yaml:22]: You are referring the SHA256 of 8.6.3.1

File: manifests/8/8x8/VirtualOfficeDesktop/8.7.2/8x8.VirtualOfficeDesktop.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{E5A87C4F-C56C-43F4-A174-E1033A02EDCD}'
+AppsAndFeaturesEntries:
+- UpgradeCode: '{66F8B350-94E0-43AF-9D26-C2385B310C96}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://vod-updates.8x8.com/ga/work-64-msi-v8.6.3-1.msi
```
Comment (shivaabhishek07) [inline manifests/8/8x8/VirtualOfficeDesktop/8.7.2/8x8.VirtualOfficeDesktop.installer.yaml:21]: You are referring the wrong InstallerUrl of 8.6.3.1 for 8.7.2 version.

**Review by shivaabhishek07 — CHANGES_REQUESTED** [review]
Please make sure you are referring the correct IndtallerUrl and InstallerSha256 of version 8.7.2

**Comment by stephengillie** [comment]
> Possible duplicate package entry.
> 
> Similar installer SHA256 hash found in manifest:
> 
> * `InstallerSha256: 81db508fee1f21d724eb8709acf79d0bd4171b72d0e3f2e02c44c150e72ae924`
>   
>   * manifests\8\8x8\VirtualOfficeDesktop\8.6.3.1

This appears to be a vanity URL. The previous version should be removed.

**Pull request closed without merging**